### DISCUSSION TITLE
VMs created by user-accounts not visible in user-accounts

### DISCUSSION THREAD

**Discussion by 1960web** [cockpit-project/cockpit#20147] [Q&A]
Hello sirs,

I'm trying to create a development environment, networked couple of computers for kids im 'trying' to mentor.

I want to have a bunch of users, each of them to have a user-account, etc

I setup Cockpit on Ubuntu 20.04 and it seems all is working fine. I setup storage pools, volumes and am able to create VMs....using my system-admin account.

Issue observed: if I use a user account (in Ubuntu) to login to cockpit, and 'Create a VM', it seems to get stuck on the 'creating........' level, and suddenly the VM disappears. The VM page even shows: 'no VM is running or defined on this host...but when I checked the storage-pools >> storage volumes >> the individual volumes are there

So Ive been googling what the issue could be.

Later, I logged back in using my Ubuntu admin account, I see that all the VMs are actually there.

Using another PC, I logged back into the server using the user-account, still the VM page shows an empty list of VMS. I attempted another VM creation, same above observation, but I can see the most recent VM listed ONLY if I login with system-admin account.
I used different browsers/computers, same observation.

Please what could be the issue.

I expected that if I have 2-users, the VMs created by each user should be listed in each account, while the system-admin account should see all VMs on the server.

Regards
1960Web

**Comment by jelly** [comment]
Are those listed VM's created as `system` or `session`? I am a bit confused how you created the VM's which are not visible under a user account, are they created by that account or not? (And by Cockpit or outside of)

**Reply by 1960web** [reply]
System

**Reply by jelly** [reply]
Then they where not created by the user you wanted too, maybe you where in administrative mode?

**Reply by 1960web** [reply]
@jelly
thanks for your response. 
No, I was in the 'user-mode'....using the user-account
are non-admin-users supposed to create vms as session?

regards
1960web
